DataAnalytics.md

```diff
@@ -0,0 +1,11 @@
+# Spark on Graviton
+Below are some general guidelines that can be referenced by users trying to improve over all Application performance across Spark clusters. Note that, there is no magic/standard values that works in all use cases and each case can be different. Most of the times, it depends on dataset sizes, cores available, workload type etc. 
```

Comment:
More introduction please about why this is difficult and what the goal should be.

The goal is generally to keep all the cpus busy and not have a small set of cores executing long running tasks. This depends on the datset size, number of instances, core count/instance, and computational complexity.

Should also have a bit of text on bottlenecks in Spark being cpu, vs disk vs. network

Reply:
Added more details in next revision